Examples/BugReport/Sources/BugReport/main.swift

```diff
@@ -0,0 +1,3 @@
+import MongoSwift
+
+print("Give us a minimal, working example of the bug you've found!")
```

Comment:
If we have `main.swift` for demonstrating the bug, what is the point of `BugReportTests.swift` (and, by extension, including Nimble)?

Reply:
let's just make them do it one way. I think main.swift is fine, we can keep it simple

Reply:
Sounds fine to me. Done.